14/umbraco-cms/reference/security/external-login-providers.md

```diff
@@ -544,12 +626,12 @@ public static class ProviderBackofficeAuthenticationExtensions
                 {
                     // The scheme must be set with this method to work for the back office
                     var schemeName =
-                        backOfficeAuthenticationBuilder.SchemeForBackOffice(GoogleBackOfficeExternalLoginProviderOptions
+                        backOfficeAuthenticationBuilder.SchemeForBackOffice(GenericOfficeExternalLoginProviderOptions
```

Comment:
getting the following error on GenericOfficeExternalLoginProviderOptions, is it the same for you?

![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/b926f509-f8c4-4963-9380-f49b7a0fbac3)